### ISSUE TITLE
anats to common modifications

### ISSUE THREAD

**Issue by nadkarni-na** [sammba-mri/sammba-mri#60]
1) expose a flag to the non-linear weight. in reality this has to be modified by the user per project because things are always different, the most common issue being colliculus coverage. the best way to use the function would be to do an initial run up to affine, modify the mask manually to make sure it roughly covers the whole brain plus some exterior tissue, then rerun up to non-linear with this manually-modified mask as the non-linear weight

2) expose a flag to use an exterior image as the first target. at the moment I guess it's the mean of brain-extracted images with their CoMs aligned

**Comment by nadkarni-na** [comment]
@salma1601 I've done (1), PR will come soon
In the mean time doesn't [this line](https://github.com/sammba-mri/sammba-mri/blob/abc5641678bab6b7975f656d758b100bae6da383/sammba/registration/struct.py#L388) just overwrite the previous result?

**Comment by salma1601** [comment]
Yes, you are right. I think the first mask is there for no reason, and should be removed.

**Comment by salma1601** [comment]
about the nonlinear bug, actually if you enter a list of [1, 2, 3, 4, 5, 6, 7] you expect that 6 qwarps are performed from inilev=0 to maxlev=1, then from 1 to 2, ...., from 6 to 7 and finally from 7 to the minimal set patch (without a max lev)?

**Comment by nadkarni-na** [comment]
see pull request #65

**Comment by salma1601** [comment]
@nadkarni-na I think this issue has been fixed by PR #65. Do you think I can close it ?

**Comment by nadkarni-na** [comment]
I haven't done (2) yet
